disable vdso loading in ELF handler unconditionally for some buggy kernel
like that shipped with RHEL(likely CentOS and SL) 5.6

  https://bugzilla.redhat.com/show_bug.cgi?id=673616

Benda Xu <heroxbd@gentoo.org> (24 Nov, 2014)

Index: work/glibc-2.20/sysdeps/unix/sysv/linux/dl-sysdep.h
===================================================================
--- work.orig/glibc-2.20/sysdeps/unix/sysv/linux/dl-sysdep.h
+++ work/glibc-2.20/sysdeps/unix/sysv/linux/dl-sysdep.h
@@ -23,7 +23,7 @@
    machines, we should look at it for unwind information even if
    we aren't making direct use of it.  So enable this across the board.  */
 
-#define NEED_DL_SYSINFO_DSO	1
+#define NEED_DL_SYSINFO_DSO	0
 
 
 #ifndef __ASSEMBLER__
